### DISCUSSION TITLE
How to use BrushTransition

### DISCUSSION THREAD

**Discussion by H0tCh0colat3** [AvaloniaUI/Avalonia#6317] [Q&A] [answered]
I see that a way to animate color supposedly exists per [Issue 3210](https://github.com/AvaloniaUI/Avalonia/issues/3210) but I'm unable to actually use it. Any attempt to use it anywhere results in "Unable to resolve type BrushTransition from namespace https://github.com/avaloniaui". I've tried using it in transitions for a window, user control, style, a completely fresh project, set nuget to use the bleeding edge build, and nothing. I can't even find the type in code behind. It appears to work just fine in the RenderDemo project.

Here is the syntax I'm using:
`
<Setter Property="Transitions">
    <Transitions>
            <BrushTransition Property="Background" Duration="0:0:0.5" />
    </Transitions>
</Setter>
`

Is there something else I need to do? Do I have the syntax wrong?

**Comment by maxkatz6** [comment, marked as answer]
It wasn't yet releases in stable version. Will be available with next one in next weeks.

> set nuget to use the bleeding edge build

Have you set nightly version though?

**Reply by H0tCh0colat3** [reply]
I followed the instructions on [this page](https://github.com/AvaloniaUI/Avalonia/wiki/Using-nightly-build-feed) and set the version to latest prerelease. Do I need to use the all builds feed instead?

Here's my nuget.config file:
![nuget config](https://user-images.githubusercontent.com/32641956/127409870-4aea2ffe-94d5-4170-a252-df0b884cf9a6.png)

and the versions that the package manager reports I have installed:
![version](https://user-images.githubusercontent.com/32641956/127409875-b02b408e-5a92-45f6-a46a-a6b63fb67932.PNG)

**Reply by maxkatz6** [reply]
Try to use "all builds" feed from here https://github.com/AvaloniaUI/Avalonia/wiki/Using-nightly-build-feed#all-builds-feed

**Reply by H0tCh0colat3** [reply]
~I'm sorry, I don't quite understand how to get packages from the all builds feed. Is there a more detailed tutorial I can reference?~

Edit: I have the nuget source working in visual studio, where do I go to do this part:
> check the build number from the build on Azure Pipelines

**Reply by maxkatz6** [reply]
Last build is cibuild0015105

You can open Azure build page from commit/PR and on PublishBuildArtifacts there will be a number:
https://dev.azure.com/AvaloniaUI/AvaloniaUI/_build/results?buildId=15105&view=logs&j=2d2b3007-3c5c-5840-9bb0-2b1ea49925f3&t=f286c637-808f-5cb6-c5ae-85afefaf805b

**Reply by H0tCh0colat3** [reply]
This works. Thank you.
